app/helpers/youtube_helper.rb

```diff
@@ -1,8 +1,25 @@
 module YoutubeHelper
+  # Look below for expanded code
+  class << YoutubeHelper
+    include YoutubeHelper
+    include ActionView::Helpers::TagHelper
+  end
+
   # the embed method will accept a regular youtube url and convert it into an embed url
-  # In our view, we can just call <%= embed(some youtube url) %>
+  # In our view, we can just call <%= YoutubeHelper.embed(some youtube url) %>
   def embed(youtube_url)
     youtube_id = youtube_url.split("=").last
     content_tag(:iframe, nil, src: "//www.youtube.com/embed/#{youtube_id}")
   end
 end
+
+# #equiv to:
+# YoutubeHelper.extend(YoutubeHelper)
+#
+# #equiv to:
+# extend self
+#
+# #equiv to:
+# YoutubeHelper.singleton_class.class_eval do
+#   include YoutubeHelper
+# end
```

Comment:
Do we need these comments in the app? I'm find with the comment on line 9, but the rest of them just look like personal notes?